packages/core/core/src/BundleGraph.js

```diff
@@ -726,35 +726,42 @@ export default class BundleGraph {
 
       // Check that every parent bundle has a bundle group in its ancestry that contains the asset.
       return parentBundleNodes.every(bundleNode => {
-        let inBundle = false;
+        if (bundleNode.type === 'root') {
+          return false;
+        }
 
+        let isReachable = true;
         this._graph.traverseAncestors(
           bundleNode,
           (node, ctx, actions) => {
+            // If we've reached the root or a context change without
+            // finding this asset in the ancestry, it is not reachable.
+            if (
+              node.type === 'root' ||
```

Comment:
Are all entry bundles attached to the root or is it possible for an entry bundle to be elsewhere in the graph? Should we check `node.value.isEntry`?

Reply:
I'm not aware of any case where that would happen, but that seems fine to me.